examples/dingo-gov-lens/static/styles.css

```
@@ -0,0 +1,436 @@
+  --blink-navy-top: #0f3f8a;
+  --text: #e6e7f4;
+  --muted: #9b9b9c;
+  --soft-text: #c6d6f8;
+  --panel: rgba(0, 10, 27, 0.78);
+  --panel-strong: rgba(4, 18, 48, 0.86);
+  --panel-deep: rgba(0, 10, 27, 0.94);
+  --border: rgba(66, 133, 244, 0.28);
+  --border-strong: rgba(66, 133, 244, 0.45);
+  --success: #38d99e;
+  --danger: #ff7a69;
+  color: var(--text);
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)